app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -42,8 +42,20 @@ export const FormAudience = ({
 }) => {
   const config = useConfig();
 
+  // This could be improved on the GraphQL side, but right now in order
```

Comment:
It seems like we should file an issue to return values rather than labels and adjust this here. Don't think that should block r+ since you've already done the work though.

Reply:
https://jira.mozilla.com/browse/EXP-698 and tests updated